### DISCUSSION TITLE
Gutenberg version vs WordPress Block Editor version

### DISCUSSION THREAD

**Discussion by scheinercc** [WordPress/gutenberg#55147] [General]
Hi there,
Is there a way to find out which version of Gutenberg is currently running in WordPress as WordPress Block Editor?
Thanks

**Comment by bph** [comment]
Yes the list is here; 
https://developer.wordpress.org/block-editor/contributors/versions-in-wordpress/

Does that help?

**Comment by scheinercc** [comment]
Thank you Birgit! Exactly what I was after.
